packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -65,6 +66,8 @@ export default declare<State>((api) => {
           paths: [state.opts.root ?? this.cwd],
         }));
       }
+
+      this.transformCache = new WeakMap();
```

Comment:
We use this to cache transformed nodes so we don't transform again. Let me know if there is a more appropriate direction.